project/backend/src/main/java/com/google/sps/servlets/RecommendationServlet.java

```diff
@@ -39,11 +38,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       List<Map.Entry<Restaurant, Double>> sortedRestaurants =
           new ArrayList(restaurantScores.entrySet());
       sortedRestaurants.sort(Map.Entry.comparingByValue(Collections.reverseOrder()));
+      response.setContentType("application/json");
       response.getWriter().println(gson.toJson(sortedRestaurants));
     } catch (JSONException e) {
       LOGGER.log(Level.WARNING, "Error parsing JSON: " + e.getMessage());
-      // TODO: handle case where list of restaurants is empty.
-      response.getWriter().println();
+      response.setContentType("text/html");
+      response.getWriter().println("Oops! We could not process your request at this time.");
```

Comment:
hmm HttpServletResponse has a sendError method we should probably use: https://javaee.github.io/javaee-spec/javadocs/javax/servlet/http/HttpServletResponse.html#sendError-int-java.lang.String- 
I think this would be SC_INTERNAL_SERVER_ERROR https://javaee.github.io/javaee-spec/javadocs/javax/servlet/http/HttpServletResponse.html#SC_INTERNAL_SERVER_ERROR

Reply:
Would I still want to log the error with Logger?

Reply:
I think we can remove that now that its been handled with sendError